include/xalloc.h

```diff
@@ -8,89 +8,101 @@
  */
 
 #ifndef UTIL_LINUX_XALLOC_H
-#define UTIL_LINUX_XALLOC_H
+# define UTIL_LINUX_XALLOC_H
 
```

Comment:
Do we really need to change indention due to #ifndef UTIL_LINUX_*_H ? I think in all others header files we ignore these special (initial) ifdef for indention. And it seems it's pretty common tradition in C, see for example /usr/include/stdlib.h.